packages/amplify-e2e-core/src/categories/storage.ts

```diff
@@ -381,12 +381,8 @@ export function addS3AndAuthWithAuthOnlyAccess(cwd: string, settings: any): Prom
       .sendCarriageReturn() // Default name
       .wait('Who should have access')
       .sendCarriageReturn() // Auth users only
-      .wait('What kind of access do you want for')
-      .send(' ')
-      .sendKeyDown()
-      .send(' ')
-      .sendKeyDown()
-      .send(' ')
+      .wait('What kind of access do you want for Authenticated users?')
+      .sendCtrlA()
```

Comment:
is this standard? I've not seen sendCtrlA() elsewhere

Reply:
Yes , this will work. I have changed it for all s3 tests. Its more reliable than "space"+ "keydown"